admin/startstop.html.md.erb

```diff
@@ -147,6 +147,12 @@ For best results in using `hawq start` and `hawq stop` to manage your HAWQ syste
     $ hawq stop master -M fast
     $ hawq stop master -M immediate
     ```
+-   If you want to reload server parameter setting on a HAWQ database where there are active connections, use the command:
```

Comment:
Small edit:

If you want to reload server parameter settings on a HAWQ cluster that has active connections